docs/src/howto-guides/023_smartrest_templates.md

```diff
@@ -0,0 +1,76 @@
+# How to use C8Y Custom SmartRest 2 Templates
+
+[Custom SmartRest Templates](https://cumulocity.com/guides/reference/smartrest-two) can be used to extend the functionality of a device to support more operations than what the [static SmartREST templates](https://cumulocity.com/guides/reference/smartrest-two/#mqtt-static-templates) offer.
+
+thin-edge.io supports subscription to custom templates as documented [here](https://cumulocity.com/guides/users-guide/device-management/#smartrest-templates).
+
+For every template that the device uses, it must publish all data to `s/uc/<template-name>` topic and subscribe to `s/dc/<template-name>` to receive data from the cloud, based on that template.
+When these templates are configured with thin-edge.io, subscriptions to all these relevant topics on Cumulocity cloud will be done by thin-edge internally. Local processes on the device can access these templates on the local MQTT broker by simply publishing to `c8y/s/uc/<template-name>` and subscribing to `c8y/s/dc/<template-name>` topics (note the `c8y/` prefix in topics).
+
+A template named `$TEMPLATE_NAME` requires the following subscriptions to be added when connecting to Cumulocity:
+
+````plain
+s/dc/$TEMPLATE_NAME
+s/uc/$TEMPLATE_NAME
+``
```

Comment:
The format of this block seems broken.
Line 12 has four backticks and line 15 has two.